modules/homefinder/widgets/class-affiliate-search-widget.php

```diff
@@ -0,0 +1,208 @@
+		echo '</p>';
+
+		// Nearby Area 1.
+		echo '<p>';
+		echo '	<label for="nearby-area-1" class="title-label">' . __( 'Nearby Area 1:', 're-pro' ) . '</label>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_city_1' ) . '" name="' . $this->get_field_name( 'nearby_city_1' ) . '" value="' . $nearby_city_1 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'City', 're-pro' ) . '</span>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_state_1' ) . '" name="' . $this->get_field_name( 'nearby_state_1' ) . '" value="' . $nearby_state_1 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'State', 're-pro' ) . '</span>';
+		echo '</p>';
+
+		// Nearby Area 2.
+		echo '<p>';
+		echo '	<label for="nearby-area-2" class="title-label">' . __( 'Nearby Area 2:', 're-pro' ) . '</label>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_city_2' ) . '" name="' . $this->get_field_name( 'nearby_city_2' ) . '" value="' . $nearby_city_2 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'City', 're-pro' ) . '</span>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_state_2' ) . '" name="' . $this->get_field_name( 'nearby_state_2' ) . '" value="' . $nearby_state_2 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'State', 're-pro' ) . '</span>';
+		echo '</p>';
+
+		// Nearby Area 3.
+		echo '<p>';
+		echo '	<label for="nearby-area-3" class="title-label">' . __( 'Nearby Area 3:', 're-pro' ) . '</label>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_city_3' ) . '" name="' . $this->get_field_name( 'nearby_city_3' ) . '" value="' . $nearby_city_3 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'City', 're-pro' ) . '</span>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_state_3' ) . '" name="' . $this->get_field_name( 'nearby_state_3' ) . '" value="' . $nearby_state_3 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'State', 're-pro' ) . '</span>';
+		echo '</p>';
+
+		// Nearby Area 4.
+		echo '<p>';
+		echo '	<label for="nearby-area-4" class="title-label">' . __( 'Nearby Area 4:', 're-pro' ) . '</label>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_city_4' ) . '" name="' . $this->get_field_name( 'nearby_city_4' ) . '" value="' . $nearby_city_4 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'City', 're-pro' ) . '</span>';
+		echo '	<input id="' . $this->get_field_id( 'nearby_state_4' ) . '" name="' . $this->get_field_name( 'nearby_state_4' ) . '" value="' . $nearby_state_4 . '" class="widefat">';
+		echo '	<span class="description">' . __( 'State', 're-pro' ) . '</span>';
+		echo '</p>';
+
+	}
+
+	/**
+	 * Update Widget Instance.
+	 *
+	 * @access public
+	 * @param mixed $new_instance New Instance.
+	 * @param mixed $old_instance Old Instance.
+	 * @return $instance
+	 */
+	public function update( $new_instance, $old_instance ) {
```

Comment:
__[phpmd]__
* The method update() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10.

<!-- Comment by [SideCI](https://sideci.com) -->